storybookjs/storybook
# Docs: Remove unused imports in MDX documentation

## What I did

I removed the unused import ﻿Story from a code snippet in the MDX documentation.

## How to test

1. Refer to [MDX Documentation](https://storybook.js.org/docs/react/writing-docs/mdx) and create `﻿Checkbox.mdx` .
1. Please confirm that the imported ﻿`Story` is unused and can be safely removed.

## Checklist

Skipping because it is a documentation-only fix.

      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
